server/utils/telemetry/rudder.go

```diff
@@ -0,0 +1,38 @@
+package telemetry
+
+import rudder "github.com/rudderlabs/analytics-go"
```

Comment:
Why `github.com/rudderlabs/analytics-go` is not part of `go.mod`?

Reply:
My mistake, forgot to add it to the commit.